manifest.json

```
@@ -15,7 +15,53 @@
     "react": "3.x",
     "admin": "0.x",
     "messages": "1.x",
-    "docs": "0.x"
+    "docs": "0.x",
+    "graphql": "1.x",
+    "node": "6.x"
```

Same as before, removing unused builders.

```suggestion
    "docs": "0.x"
```